src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskListTabPanel.tsx

```diff
@@ -3,28 +3,230 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Box, Link, Typography } from '@mui/material';
+import { Box, Link, useTheme } from '@mui/material';
+import { DataGrid, GridActionsCellItem, GridColumns, GridRowId, GridRowParams } from '@mui/x-data-grid';
+import { RoleEnum, Task, TaskPriority, TaskStatus, UserPreview } from 'shared';
+import { datePipe, fullNamePipe } from '../../../utils/pipes';
+import { GridColDefStyle } from '../../../utils/tables';
+import PauseIcon from '@mui/icons-material/Pause';
+import DeleteIcon from '@mui/icons-material/Delete';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import CheckIcon from '@mui/icons-material/Check';
+import { useAuth } from '../../../hooks/auth.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import React from 'react';
+import { useSetTaskStatus } from '../../../hooks/tasks.hooks';
+
+//this is needed to fix some weird bug with getActions()
+//see comment by michaldudak commented on Dec 5, 2022
+//https://github.com/mui/material-ui/issues/35287
+declare global {
+  namespace React {
+    interface DOMAttributes<T> {
+      onResize?: ReactEventHandler<T> | undefined;
+      onResizeCapture?: ReactEventHandler<T> | undefined;
+      nonce?: string | undefined;
+    }
+  }
+}
 
 interface TaskListTabPanelProps {
   index: number;
   value: number;
+  tasks: Task[];
+  status: TaskStatus;
 }
 
+type Row = { id: number; title: string; deadline: string; priority: TaskPriority; assignee: string; taskId: string };
+
 const TaskListTabPanel = (props: TaskListTabPanelProps) => {
-  const { value, index } = props;
+  const { value, index, tasks, status } = props;
+  const editTaskStatus = useSetTaskStatus();
+
+  const auth = useAuth();
+
+  const disabled = auth.user?.role === RoleEnum.GUEST;
+
+  const theme = useTheme();
+
+  const renderNotes = () => <Link>See Notes</Link>;
+
+  const moveToBacklog = React.useCallback(
+    (id: string) => async () => {
+      await editTaskStatus.mutateAsync({ taskId: id, status: TaskStatus.IN_BACKLOG });
```

Comment:
I tried testing this as a user without the correct permissions. I get a console.error saying Access Denied, but apart from this, no other indication that my action failed. Make something in the UI (toast?) to indicate that a change was not successful.